examples/format/simple/.expected/config.yaml

```diff
@@ -0,0 +1 @@
+runCount: 2
```

Comment:
@mengqiy The default value should be idempotent, you should need to set this explicitly

Reply:
Yes. This clean up should happen for all other functions.